docs/source/asr/models.rst

```diff
@@ -148,6 +148,9 @@ the transducer config at ``<NeMo_git_root>/examples/asr/conf/fastconformer/fast-
 
 Note that both configs are subword-based (BPE).
 
+You can also train these models with longformer-style attention using the following configs: CTC config at
+``<NeMo_git_root>/examples/asr/conf/fastconformer/fast-conformer-long_ctc_bpe.yaml`` and transducer config at ``<NeMo_git_root>/examples/asr/conf/fastconformer/fast-conformer-long_transducer_bpe.yaml``
+
```

Comment:
Do we have any docs on this local-attention with Longformer support?

Reply:
I added link to paper, I don't think we really need the full explanation in the docs

Reply:
I don't mean on how longformer works. I meant docs on how users may switch to longformer or use it for inference or training. Is there any place in the docs that we mention we support these feature?

Reply:
Training is here, just switching to local context is mentioned here: https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/asr/results.html#inference-on-long-audio 
Switching to local+global isn't mentioned there because it won't work without additional training

Reply:
I see, it looks fine, but shouldn't we add how to do inference with models trained with global tokens here now that we are going to add training? Or the same command works for models trained with global tokens?

Reply:
If the model is already trained with global tokens then you can do inference in the same way as any other model. Added links to the configs to that part as well